sdk/common/src/main/java/io/opentelemetry/sdk/resources/Resource.java

```diff
@@ -78,6 +80,11 @@ private static String readVersion() {
   private static Resource readResourceFromProviders() {
     Resource result = Resource.EMPTY;
     for (ResourceProvider resourceProvider : ServiceLoader.load(ResourceProvider.class)) {
+      if (RESOURCES_CONFIG
+          .getDisabledResourceProviders()
+          .contains(resourceProvider.getClass().getSimpleName())) {
```

Comment:
```suggestion
          .contains(resourceProvider.getClass().getName())) {
```

So it's fully qualified